shared-bindings/busio/SPI.c

```diff
@@ -65,7 +65,7 @@
 //|            peripherals.
 //|
 //|         .. seealso:: Using this class directly requires careful lock management.
-//|             Instead, use :class:`~adafruit_bus_device.spi_device.SPIDevice` to
+//|             Instead, use :class:`~adafruit_bus_device.SPIDevice` to
```

Comment:
I think this is not right? It's still inside `spi_device`.

Reply:
Same reply to previous comment about `i2c_device`